src/sinol_make/commands/run/__init__.py

```diff
@@ -594,8 +698,16 @@ def warn_if_not_empty(set, message):
         warn_if_not_empty(diff.removed_groups, "Groups were removed")
 
         for change in diff.changes:
-            print(util.warning("Solution %s passed group %d with status %s while it should pass with status %s." %
-                               (change.solution, change.group, change.result, change.old_result)))
+            if isinstance(change, ResultChange):
+                if isinstance(change.result, str):
+                    print(util.warning("Solution %s passed group %d with status %s while it should pass with status %s." %
+                                       (change.solution, change.group, change.result, change.old_result)))
+                elif isinstance(change.result, int):
+                    print(util.warning("Solution %s passed group %d with %d points while it should pass with %s points." %
+                                       (change.solution, change.group, change.result, change.old_result)))
+            elif isinstance(change, PointsChange):
+                print(util.warning("Solution %s passed group %d with %d points while it should pass with %d points." %
+                                   (change.solution, change.group, change.new_points, change.old_points)))
```

Comment:
Those two strings are the same, should they be? If yes, maybe we can shorten the code?

Reply:
They shouldn't be

Reply:
Sorry, they should, i shortened the code: c01a6932b6816a2df51ad94039e2eab099d99597